### PR TITLE
[castopod] Write guide to install castopod

### LINKED ISSUE #1257: [castropod] create guide
[Seems pretty straight forward](https://docs.castopod.org/getting-started/install.html)

### PR DESCRIPTION
**Pull request by Ryuno-Ki**
This will close #1257.

I have some parts marked with TODO where I would need a hand.
Also I'm not sure whether I got the formatting right all the time.
It's my first guide :tea: 

Signed-off-by: André Jaenisch <andre.jaenisch@posteo.de>

### CODE REVIEW

File: source/guide_castopod.rst [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  not in a subdirectory of a domain.
+
+Installation
+============
+
+First, go one level above your :manual:`DocumentRoot <web-documentroot>`
+and download the latest version. Make sure, it is a Castopod Package and not
+the source code. At time of this writing this is v1.0.0-beta.14.
+
+TODO: the permalink to latest version requires login:
```
Comment (noave) [inline source/guide_castopod.rst:79]: that is bad, is it possible to get a link where you just need to lookup the latest version and replace it within the link?

File: source/guide_castopod.rst [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  [isabell@stardust isabell]$ rm castopod-1.0.0-beta.14.tar.gz
+  [isabell@stardust isabell]$
+
+Remove your empty DocumentRoot and create a new symbolic link
+to the castopod directory:
+
+::
+
+  [isabell@stardust ~]$ cd /var/www/virtual/$USER
+  [isabell@stardust isabell]$ rm --force html/nocontent.html; rmdir html
```
Comment (noave) [inline source/guide_castopod.rst:109]: no need for --force here

File: source/guide_castopod.rst [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+  [isabell@stardust isabell]$ ln --symbolic /var/www/virtual/$USER/castopod/public html
+  [isabell@stardust isabell]$
+
+Update .htaccess
+----------------
+
+Open the .htaccess file in html and remove unsupported lines: All and FollowSymLinks.
+
+TODO: Codeblock needed here?
```
Comment (noave) [inline source/guide_castopod.rst:118]: you should provide the full path to `.htaccess` and in a codeblock the lines that have to be removed (better comment with `#`) and then something like "edit the file with your editor and comment these lines out"

File: source/guide_castopod.rst [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+Finishing installation
+======================
+
+Before finishing the installation via wizard, set up some cronjobs for
```
Comment (noave) [inline source/guide_castopod.rst:123]: please describe shortly how to edit cronjobs via `crontab -e`

File: source/guide_castopod.rst [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+   */10 * * * * cd /var/www/virtual/$USER/html && php index.php scheduled-activities
+
+For having your episodes be broadcasted on open hubs upon publication using WebSub:
+
+.. code-block:: none
+
+   */10 * * * * cd /var/www/virtual/$USER/html && php index.php scheduled-websub-publish
+
+Point your browser to your uberspace URL or domain.
+You will see an error page. Add ``cp-install`` to the URL and finish the
```
Comment (noave) [inline source/guide_castopod.rst:140]: just give the domain https://isabell.uber.space/cp-install and advise users to change it accordingly

File: source/guide_castopod.rst [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+Fixing file permissions
+-----------------------
+
+The permissions of the files are not correct. Follow
+:manual_anchor:`DocumentRoot <web-documentroot#permissions>` to update them.
+
+Security
+========
+
+After installation and updates, ensure that the file permissions are correct.
```
Comment (noave) [inline source/guide_castopod.rst:153]: provide the code here to do this

File: source/guide_castopod.rst [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+
+If there is a new release available, move your /var/www/virtual/$USER/castopod/.env and
+/var/www/virtual/$USER/castopod/public/media outside of those directories.
+Then delete all other files there. Follow the installation instructions from above.
+Move the files back to their original place.
+
+Sometimes there are additional update instructions you have to follow. For example, .sql
+files containing migrations.
+
+TODO: Document how to run these without phpMyAdmin!
```
Comment (noave) [inline source/guide_castopod.rst:170]: please also provide code for all moving files etc. :-)

**Review by noave — CHANGES_REQUESTED** [review]
Thanks :) I added some comments, there might be some more things to change afterwards, but lets start with these :-)

**Comment by Ryuno-Ki** [comment]
I waited so long that the Uberspace instance I used for setting it up (to verify my instructions) was disassembled by now.

**Comment by luto** [comment]
Thanks for your guide! We would like to keep the list of open pull requests clean to improve the overview. Since this has been dormant for half a year, I'm closing this now. Once you find time to look at the comments, feel free to submit it again.

**Pull request closed without merging**